sparse_strips/vello_common/src/mask.rs

```diff
@@ -25,55 +29,85 @@ impl Mask {
         Self::new_with(pixmap, false)
     }
 
+    /// Create a new mask from the given alpha data.
+    ///
+    /// The `data` vector must be of length `width * height` exactly.
+    ///
+    /// The pixels are in row-major order.
+    ///
+    /// # Panics
+    ///
+    /// Panics if the `data` vector is not of length `width * height`.
+    pub fn from_parts(data: Vec<u8>, width: u16, height: u16) -> Self {
+        assert_eq!(
+            data.len(),
+            usize::from(width) * usize::from(height),
+            "Expected `data` to have length of exactly `width * height`"
```

Comment:
```suggestion
            "`data` should have `width * height` length"
```

Saves a few bytes in the binary